helm/cluster/templates/clusterapi/workers/_helpers_flatcar.tpl

```diff
@@ -0,0 +1,37 @@
+{{- define "cluster.internal.workers.kubeadm.ignition" }}
```

Comment:
Worker-specific named templates for Flatcar systemd and storage config (these templates here just include generic templates and pass values for workers).